src/adapters/rubicon.js

```diff
@@ -97,7 +97,8 @@ var RubiconAdapter = function RubiconAdapter() {
 
 	}
 
-	pbjs.handleRubiconCallback = function(response) {
+	window.pbjs = window.pbjs || {que: []};
```

Comment:
Good catch. Could you refactor this into the first line of src/prebid.js - since this is an issue with pbjs being undefined other adapters could also have this issue. Thanks!